app/features/analysis/calculate/OneRMCalculateActions.js

```diff
@@ -167,8 +169,12 @@ const logCalculate1RMAnalytics = (state, exercise, numIncludeTags, numExcludeTag
         slope: slope,
         min_weight: minWeight,
         max_weight: maxWeight,
+        min_lb_weight: WeightConversion.weightInLBs(metric, minWeight),
+        max_lb_weight: WeightConversion.weightInLBs(metric, maxWeight),
+        weight_lb_range: WeightConversion.weightInLBs(metric, (maxWeight - minWeight)),
```

Comment:
- [ ] technically, if max and min are KG and LB respectively, you're mathing two different unit types and THEN converting it, which is wrong

Reply:
How would max and min ever be different metrics?